src/components/LoginPanel.tsx

```diff
@@ -1,23 +1,27 @@
 "use client";
 
-import { currentTheme } from "@/app/themes";
 import { DatabaseAutoType } from "@/supabase";
 import { createClientComponentClient } from "@supabase/auth-helpers-nextjs";
 import { Auth } from "@supabase/auth-ui-react";
 import { ThemeSupa } from "@supabase/auth-ui-shared";
 import React from "react";
-import styled from "styled-components";
+import styled, { useTheme } from "styled-components";
 
 const MiddleDiv = styled.div`
     max-width: 400px;
-    border: solid 1px black;
+    border: solid 1px ${(props) => props.theme.foregroundColor};
     border-radius: 10px;
     padding: 50px;
     margin: auto;
+
+    & button:hover {
+        color: ${(props) => props.theme.secondaryForegroundColor};
```

Comment:
this isn't needed - use the `defaultButtonBackgroundHover` attribute in the `colors` object
(https://github.com/supabase/auth-ui/blob/main/packages/shared/src/theming/Themes.ts)

Reply:
This is a `color` and not a `background-color` so this is still required to pass accessibility checks.

Reply:
whyyyy isn't there a `defaultButtonHoverColor`...